src/cgame/cg_draw.c

```diff
@@ -100,6 +100,50 @@ int CG_Text_Width_Ext(const char *text, float scale, int limit, fontHelper_t *fo
 	return out * scale * Q_UTF8_GlyphScale(font);
 }
 
+/**
+ * @brief CG_Text_Width_Ext_Float
+ * @param[in] text
+ * @param[in] scale
+ * @param[in] limit
+ * @param[in] font
+ * @return
+ */
+float CG_Text_Width_Ext_Float(const char *text, float scale, int limit, fontHelper_t *font)
```

Comment:
Better to avoid direct duplicates, as it adds unnecessary strain to support.
One simple solution to this problem would be calling `CG_Text_Width_Ext_Float` from `CG_Text_Width_Ext` and casting the result to int.

```c
int CG_Text_Width_Ext(const char *text, float scale, int limit, fontHelper_t *font)
{
    return CG_Text_Width_Ext_Float(text, scale, limit, font);
}
```

I think compiler in the end will do the right thing.

Reply:
Done.